src/app.controller.ts

```diff
@@ -164,4 +205,24 @@ export class AppController {
     );
     res.json(result.data).send();
   }
+
+  @Post('/schema/upload')
+  @UseInterceptors(FileInterceptor('upload'))
+  async uploadSchema(@UploadedFile() file: Express.Multer.File, @Res() res) {
+    console.log(file);
+    await fs.writeFile(join('schema', file.originalname), file.buffer);
+    res.status(302).redirect('/api/issuer');
+  }
```

Comment:
Since the functionality works via the UI and the failed upload is most likely caused by a local problem, this should not prevent us from merging this to main.